syntax = "proto3";

package acrux;

message Server {
    string name = 1;
    string version = 2;
}

service AFLPP {
  rpc start (StartRequest) returns (StartReply) {}
  rpc stop (StopRequest) returns (StopReply) {}
}


message StartRequest {
  bytes binary = 1;
  string binary_args = 2;
  string aflpp_args = 3;
  repeated bytes seeds = 4;
}


message StartReply {
  bool success = 1;
}


message StopRequest {}


message StopReply {
  bool success = 1;
}
